storage-s3-dynamodb/src/main/java/io/delta/storage/BaseExternalLogStore.java

```diff
@@ -311,15 +356,18 @@ private void fixDeltaLog(FileSystem fs, ExternalCommitEntry entry) throws IOExce
     }
 
    /**
-    * Copies file within filesystem
+    * Copies file within filesystem.
+    *
     * @param fs reference to [[FileSystem]]
     * @param src path to source file
     * @param dst path to destination file
     */
     private void copyFile(FileSystem fs, Path src, Path dst) throws IOException {
-        LOG.debug("copy file: {} -> {}", src, dst);
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("copy file: {} -> {}", src, dst);
+        }
         FSDataInputStream input_stream = fs.open(src);
-        FSDataOutputStream output_stream = fs.create(dst, true);
+        FSDataOutputStream output_stream = fs.create(dst, false); // overwrite=false
```

Comment:
Do we have any place converting `org.apache.hadoop.fs.FileAlreadyExistsException` thrown from here to `DeltaErrors.fileAlreadyExists`?

Reply:
No. `delta-core` depends on `delta-storage`. `storage-s3-dynamodb` also depends on `delta-storage`, it can't access code from `delta-core`.

But we should convert this error here it to `java.nio.file.FileAlreadyExistsException`